src/vma/sock/sock-redirect.cpp

```diff
@@ -1779,8 +1778,8 @@ inline ssize_t sendfile_helper(socket_fd_api* p_socket_object, int in_fd, __off6
 {
 	__off64_t orig = 0;
 	iovec piov[1];
-	char buf[SENDFILE_BUFFER_SIZE];
-	ssize_t toRead, numRead, numSent, totSent = 0;
+	char buf[count];
```

Comment:
Please consider an alternative to VLA usage. It comes from C99 and is not part of C++ standard as I remember (it means potential issues in different compilers). See also http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3810.pdf and https://stackoverflow.com/questions/1887097/why-arent-variable-length-arrays-part-of-the-c-standard.
I understand that there are some places in our code with such things but probably it is not good to add new.

Reply:
:+1: